docs/source/glm.rst

```diff
@@ -180,11 +184,8 @@ available link functions can be obtained by
    NegativeBinomial
    Power
    cauchy
-   cloglog
    identity
    inverse_power
    inverse_squared
-   log
-   logit
```

Comment:
I think those are needed, but they only work on case sensitive operating systems, i.e. not on Windows